tests/resources/integration_test_data/error_triggering_asset_ref/parking_lot.usd

```diff
@@ -3,7 +3,7 @@
 def "ParkingLot"
 {
     def "ParkingLot_Floor_1" (
-        references = @bal:///@</ParkingLot_Floor>
+        references = @bal:///doesntexist@</ParkingLot_Floor>
```

Comment:
I'm unsure what this change means.

Couldn't spot anything mentioned in 1583204615fbc82d502a166bac38155d778a6052

Reply:
It's part of `Make the tests a bit more agnostic to the particular manager` :slightly_smiling_face: .  

The original `bal:///` triggers a "malformedEntityReference" error in BAL but doesn't trigger an error in SimpleCppManager, so I changed it to a non-existent reference to instead trigger a "entityResolutionError".

I'll add some more detail to the commit message.

Reply:
I've updated the code comment explaining the related test in bb426220 - the commit message will be rolled into the target commit on squash/rebase.